.circleci/config.yml

```diff
@@ -137,17 +137,20 @@ jobs:
             - image.tar
 
   build_dev_image:
-    executor: machine_image
+    docker:
+      - image: docker:18.09
     steps:
       - checkout
+      - setup_remote_docker:
+          docker_layer_caching: false
       - run: |
-          set -x
+          apk add make bash git
+          git checkout -b $CIRCLE_BRANCH
```

Comment:
Since the docker image doesn't include git, we have to install git for the changes-include-dir script.  Also, executing `git diff` without the `git checkout` returns:
```
Not currently on any branch.
nothing to commit, working tree clean
```
So I used `git checkout` to simply reattach the local repo to HEAD.